docs/kb/types.md

```diff
@@ -0,0 +1,58 @@
+---
+order: 6
+description: Reference list of supported types. 
+---
+
+# Starport Supported Types
+
+Types with CRUD operations are scaffolded with the `starport scaffold` command. 
+
+## Built-in Types
+
+| Type         | Alias    | Index | Code Type   | Description                     |
+| ------------ | -------- | ----- | ----------- | ------------------------------- |
+| string       | -        | yes   | string      | Text type                       |
+| array.string | strings  | no    | []string    | List of text type               |
+| bool         | -        | yes   | bool        | Boolean type                    |
+| int          | -        | yes   | int32       | Integer type                    |
+| array.int    | ints     | no    | []int32     | List of integers types          |
+| uint         | -        | yes   | uint64      | Unsigned integer type           |
+| array.uint   | uints    | no    | []uint64    | List of unsigned integers types |
+| coin         | -        | no    | sdk.Coin    | Cosmos SDK coin type            |
+| array.coin   | coins    | no    | sdk.Coins   | List of Cosmos SDK coin types   |
+
+You cannot use some types as an index, like the map and list indexes and module params.
```

Comment:
```suggestion
**Note:** Map, list indexes, and module params cannot be used as an index.
```

Reply:
@Pantani let's be clear on which types can be used as an index (the command descriptions maybe?) or is this good?

Reply:
I added it to the table. There is a section called `Index`